### ISSUE TITLE
pep8ify won't run multiple times

### ISSUE THREAD

**Issue by sauln** [pep8speaks-org/pep8speaks#88]
First off, thank you for this great tool! I've been using pep8speaks and it's automatic pull requests in all my projects.

Most recently, I've been working on a PR for a while and have tried using the pep8ify command multiple times, but it hasn't worked.  

### Steps
Initially, I issued the PR and added a comment `@pep8speaks pep8ify`. I then merged the suggested changes.  A few days later, after more commits were added, pep8speaks updated correctly.  Then I tried issuing the command `@pep8speaks pep8ify` again so pep8speaks would create another PR.  

### Result
It looks like, pep8speaks made a fork of the library, but never issued a pull request. You can see here pep8speaks@5cfadbc1c39494ab2d762afaeaa6bc982fb1d5ee that it even made the commits with the changes.

The PR I've been working on is mlwave/kepler-mapper#77.  I have manually created a new pull request from pep8speak's fork.

**Comment by OrkoHunter** [comment]
Hello @sauln! Thank you for trying out PEP8Speaks!
I can confirm that this is clearly an issue since pep8ify is currently designed to be used only once per PR. Looking at the case of `kepler-mapper`, I think it should be redesigned to be called multiple times. Unfortunately, I don't seem to find much time to work on this in near future.

**Comment by sauln** [comment]
Hi @OrkoHunter. Thanks for getting back to me.  I love PEP8Speaks! 

As a temporary work around, I've found that if I delete my original comment calling pep8ify and delete pep8speaks' comment "Here you go...", then it will issue a second pull request.
